### DISCUSSION TITLE
Could someone teach me how to fix the connection bar when using full screen?

### DISCUSSION THREAD

**Discussion by JiaxinWang-thu** [1Remote/1Remote#375] [Q&A] [answered]
Could someone teach me how to fix the connection bar when using full screen?

I have chosen the item -- "display the connection bar when use full screen". However, the connection bar cannot be fixed on the top of my screen. It automatically disappears.

**Comment by majkinetor** [comment, marked as answer]
CTRL ALT PAUSE

https://stackoverflow.com/questions/295211/keyboard-shortcut-to-move-away-from-a-full-screen-remote-desktop-session

**Reply by VShawn** [reply]
I remember there was a pin button on connection bar

**Reply by JiaxinWang-thu** [reply]
Thanks for your solutions. CTRL + ALT +HOME is also useful to make the bar reapper. And I know there is a pin button, which also helps.
However, does it possible to set the bar fixed automatically upon entering the remote desktop?

**Reply by VShawn** [reply]
unfortunately, we are not able to set it to fixed, MS didn't offer a interface for this.

**Reply by JiaxinWang-thu** [reply]
Thanks.

**Reply by VShawn** [reply]
Oh I find how to fix the bar automatically. create a ticket for it: #380
